module/icp/io/skein_mod.c

```diff
@@ -231,6 +231,19 @@ skein_mod_init(void)
 
 int
 skein_mod_fini(void) {
+	int ret;
+
+	if (skein_prov_handle != 0) {
+		if ((ret = crypto_unregister_provider(skein_prov_handle)) !=
+		    CRYPTO_SUCCESS) {
```

Comment:
[style] This one's situation dependent but often it's clearer to pull the assignment out of the conditional in cases like this where the line wraps.

```
		error = crypto_unregister_provider(skein_prov_handle);
		if (error != CRYPTO_SUCCESS) {
		}
```